### ISSUE TITLE
allow to `xst package install` to fetch packages from a repository (instead of the instance)

### ISSUE THREAD

**Issue by line-o** [eXist-db/xst#80]
### Enhancement

This might be handy for instances who do not have access to any package repository.
`xst` would act as a proxy, fetching the package then uploading and installing it in the instance that is connected.
  

### Alternatives

Not doing implementing this feature, because it is not worth it.

**Comment by joewiz** [comment]
Could you describe the current behavior? What does xst currently treat as its package repository?

**Comment by joewiz** [comment]
Is this related to https://github.com/eXist-db/xst/issues/13 ?

**Comment by line-o** [comment]
xst does not have a package repository but tells the exist-db instance it is connected to which repository to use for dependency resolution.

**Comment by line-o** [comment]
@joewiz yes, both issues are related insofar that they allow to specify a repository other than the public one on exist-db.org.
This feature here would be interesting if the instance itself cannot access any repository. Here, xst would do the package fetching and then upload the package(s) and call install-and-deploy-from-db. Naturally, you would have to care for those to be installed so that all dependencies are met at installation time.

**Comment by line-o** [comment]
Pretty much like the autodeploy folder works.
